File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        ref = [0] * 26
        for i in s:
            ref[ord(i)-ord('a')] += 1
        for j in t:
            if ref[ord(j)-ord('a')] <= 0:
                return False
            ref[ord(j)-ord('a')] -= 1
        return sum(ref) == 0

File: 0242-valid-anagram/0242-valid-anagram.go
func isAnagram(s string, t string) bool {
    ref := [26]int{}
    for _, r := range s {
        ref[r - rune('a')]++
    }
    for _, r := range t {
        ref[r - rune('a')]--
    }
    
    return ref == [26]int{}
}